spec/index.html

```diff
@@ -820,20 +820,42 @@ <h3><code>rdf:nil</code></h3>
 
 <section id="ch_ttrei">
   <h2>Triple-term based reification</h2>
-  <p>A triple term is a construct defined in the RDF 1.2 Concepts and Abstract 
-    Syntax specification [[RDF12-CONCEPTS]], referring to an RDF triple that 
-    is used as a value, specifically, in the object position of another triple. 
-    Importantly, a triple term does not imply assertion; that is, the triple it 
-    denotes is not necessarily asserted in the RDF graph. This enables the 
-    representation of metadata or commentary about triples that may or may not 
-    be part of the asserted graph content, including potentially contradictory 
-    relationships. For a triple term to be considered an asserted triple, it 
-    must also appear explicitly in an RDF graph as a top-level triple. Otherwise, 
-    it remains unasserted and serves primarily as a reference or subject of metadata.</p>
-  <p>A triple term can be the object of a triple whose predicate is 
+  <p>
+    A triple term is a construct defined in [[RDF12-CONCEPTS]], referring to an RDF triple that is used 
+    as a value, specifically, in the object position of another triple. This use represents a reference to a proposition, 
+    and, importantly, a triple term does not by itself imply assertion; that is, the proposition it denotes 
+    is not necessarily a fact asserted by the RDF graph. This is only the case if that triple is also 
+    asserted in the RDF graph — that is, if it is a direct element of that set of triples.
+    Otherwise, it remains unasserted and serves primarily as a reference or as the subject of metadata. 
+    This enables the representation of metadata or commentary about triples that may or may not be part 
+    of the asserted graph content, including potentially contradictory relationships.
+  </p>
+
+  <p>
+    RDF terms that appear in a triple term have the same denotation as when they appear in an asserted 
+    triple in the graph. For this reason, triple terms are said to be <em>transparent</em>.
+  </p>
+
+  <p>
+    A triple term can be the object of a triple whose predicate is 
     <a href="#ch_reifies"><code>rdf:reifies</code></a>. In such cases, the resulting triple is referred 
-    to as a reifying triple, and its subject is termed a reifier. The reifier 
-    can be used to make further statements about the triple term.</p>
+    to as a <em>reifying triple</em>, and its subject is termed a <em>reifier</em>. 
```

Comment:
To make sure this isn't seem as a different definition, the `<em>` can be links to the point of definition in RDF Concepts.

```suggestion
    to as a <em><a data-cite="RDF-CONCEPTS12#dfn-reifying-triple>reifying triple</a></em>, and its subject is termed a <em><a data-cite="RDF-CONCEPTS12#`#dfn-reifier>reifier</a></em>. 
```